relion/_parser/ctffind.py

```diff
@@ -1,7 +1,24 @@
+from gemmi import cif
+from pathlib import Path
```

Comment:
`self._basepath` is already guaranteed to be of type `Path`, so we don't need to import from `pathlib` and don't need to create `Path` objects again

Reply:
I have changed this now, but the self._basepath becomes a MotionCorr object after the __init__ method and I don't understand why.

Reply:
I thought it might be the caching but it happens even without the @functools decorator

Reply:
Actually I see 
`@property
 @functools.lru_cache(maxsize=1)
 def motioncorrection(self):
     return MotionCorr(self.basepath / "MotionCorr")`

- Why does this need to return a MotionCorr object rather than just self.basepath / "MotionCorr" ?

Reply:
I think I understand this now - its kind of a shortcut